src/testdrive/src/main.rs

```diff
@@ -193,6 +202,9 @@ async fn main() {
         args.materialized_url.get_hosts()[0],
         args.max_errors
     );
+    if let (Some(shard), Some(shard_count)) = (args.shard, args.shard_count) {
+        println!("    shard: {}/{}", shard, shard_count);
```

Comment:
This prints 0/4, 1/4, 2/4 , 3/4 :-) I think it is OK .

Reply:
Hah, I noticed that too! I'll fix.